packages/gateway/src/graphql/schema.d.ts

```diff
@@ -1221,6 +1234,16 @@ export const enum GQLRegStatus {
   ISSUED = 'ISSUED'
 }
 
+export interface GQLIdentifierInput {
+  type: string
+  value: string
+}
+
+export const enum GQLRejectionReason {
+  other = 'other',
+  duplicate = 'duplicate'
+}
```

Comment:
As far as I can see, [this frontend code](https://github.com/opencrvs/opencrvs-core/blob/8a6afc7b41bb777aeb951e0ea11c607b2c506fc0/packages/client/src/views/SearchResult/SearchResult.tsx#L89-L102) acts as a source of truth for the rejection reasons... this will cause issues in the future unless we refactor the reasons to be strictly typed.